processors/crownandcaliber_processor.rb

```diff
@@ -5,50 +5,28 @@
 require_relative '../parsers/crownandcaliber_parser'
 
 class CrownandcaliberProcessor < BaseProcessor
-  attr_accessor :page_content, :page
+  PAGE_TAG = '.card-title.ng-binding'
+  ITEM_TAG = '.vendor'
 
-  HOST = 'https://www.crownandcaliber.com/'
+  PROTOCOL = 'https:'
+  HOST = '//www.crownandcaliber.com/'
```

Comment:
why URI string consist `//` prefix? Remove it

Reply:
and what for you use `PROTOCOL` and `HOST` separately

Reply:
Because I get items links in next format:
![Screenshot from 2022-03-14 09-53-16](https://user-images.githubusercontent.com/94611266/158137817-4059b27f-0c0c-445c-a64c-ad040e1b04af.png)
And I need to add 'https:' to visit these links and parse the information from them